sql/src/test/java/org/apache/druid/sql/calcite/CalciteArraysQueryTest.java

```diff
@@ -143,54 +144,6 @@ public void testSelectNonConstantArrayExpressionFromTable() throws Exception
     );
   }
 
-  @Test
-  public void testSelectNonConstantArrayExpressionFromTableForMultival() throws Exception
```

Comment:
Why did you delete this test?

Reply:
heh, i wrote this test originally to document moderately strange current behavior. This PR changes the behavior of `array` constructor function to no longer "map" multi-value string inputs and instead treat them as arrays, so it should probably be repurposed to test whatever the new behavior should be probably

The map behavior was not very intuitive, since it would make a nested array where the elements are single element arrays from the multi-value string, so idk that it needs preserved.

We might want to consider making use of a multi-value string inside of the array constructor an error, since i'm not sure there is an intuitive behavior because it is ambiguous whether the user is acknowledging the column as a multi-value, or doesn't know and thinks it single valued, which most of the other functions don't have this ambiguity, though it probably doesn't need to be changed in this PR.